### PR TITLE
msx2_flop.xml: Added 54 items, 49 working. Replaced 1 item. Removed 5 items.

### PR DESCRIPTION
**Pull request by wilbertpol**
Replaced with a better dump: VS Rotation (Japan) [file-hunter]
Removed Ultima IV - Quest of the Avatar (Japan, alt disk 2) (disk 2 is from an English translation) 
Removed Vectron (Netherlands) and Vectron (Netherlands, alt) (extracted from a compilation) 
Removed Zoo (Netherlands, alt) and Zoo (Netherlands, alt 2) (hacked versions)

New working software list items
-------------------------------
Konami Game Collection Bangai-hen (Japan, alt) [file-hunter] 
The Legend of Shonan (Japan) [file-hunter]
Sailor-fuku Senshi Felis (Japan) [file-hunter]
Tempo Typen (Netherlands) [file-hunter]
Tenkyuhai Special - Tougen no Utage (Japan) [file-hunter] 
Tenkyuhai Special - Tougen no Utage II (Japan) [file-hunter] 
Thanatos (Japan) [file-hunter]
Tokimeki Sports Gal (Japan) [file-hunter]
Tominaga Koukou Tantei-bu (Japan) [file-hunter]
Trilogy Kuki Youka Shinden (Japan) [file-hunter]
The Tucs (Japan) [file-hunter]
Tulip Ichigou (Japan) [file-hunter]
Ultima II - The Revenge of the Enchantress (Japan) [file-hunter] 
Undead Line (Woomb) [file-hunter]
Wizardry Scenario #*3* - The Legacy of Llylgamyn (Japan) [file-hunter] 
Xak - The Art of Visual Stage (Japan, Woomb) [file-hunter] 
Yoshida Koumuten Data Library Vol. 2 (Japan) [file-hunter] 
Yoshida Koumuten Data Library Vol. 3 (Japan) [file-hunter] 
Yume Pro RPG Shaon-ban (Japan) [file-hunter]
Yumeji Asakusa-Kitan (Japan) [file-hunter]
Yuurei-kun (Japan) [file-hunter]
Zoo (Europe) [file-hunter]
GAME100 (Japan) [file-hunter]
Go! Volcano [NAGI-P SOFT]
Las Aventuras de Rudolphine Rur (Spanish) [Dwalin] 
MSX Light [MSXdev]
Siege [NAGI-P SOFT]
Teddy's in Action Part 2 [file-hunter]
Terrahawks [file-hunter]
Tetravex (Netherlands) [file-hunter]
Tetris Master (Japan) [file-hunter]
Tetris Master - Operation Maison Ikkoku (Japan) [file-hunter] 
Tetris Master - Operation Orange Road (Japan) [file-hunter] 
Tetris Master - Operation Ranma 1/2 (Japan) [file-hunter] 
Tetris Master - Series 1 Ranma 1/2 (Japan) [file-hunter] 
Thunderbirds are Go (Netherlands, promo) [file-hunter] 
Thunderbirds to the Rescue (Netherlands, promo) [file-hunter] 
Tile Golf [NAGI-P SOFT]
Triplex (Netherlands) [file-hunter]
Trivial Pursuit (Netherlands) [file-hunter]
Trivial Pursuit - Aanvulling Jaareditie 1995 (Netherlands) [file-hunter] 
Tunez: Garfield Edition [file-hunter]
The UHF Painter (Italy) [file-hunter]
War World FM-PAC Demo (Netherlands) [file-hunter]
Wiz Kids (Japan) [file-hunter]
Yupipati (demo) [file-hunter]
Zombie Night [Alberto Sgaggero]
Zoo Rally (Russia) [file-hunter]
Zoto (Germany?) [file-hunter]

New NOT_WORKING software list additions
------------------------------------------
HBI-V1 Video Digitizer (Japan) [file-hunter]
Himitsu no Hanazono (Japan) [file-hunter]
Veldslag (Netherlands) [file-hunter]
Zeeslag (Netherlands) [file-hunter]
Zeeslag (Netherlands, demo) [file-hunter]

### CODE REVIEW

File: hash/msx2_flop.xml [hunk 1]
```diff
@@ -7137,6 +7137,18 @@ The following floppies came with the machines.
...
+		<description>HBI-V1 Video Digitizer (Japan)</description>
+		<year>1989</year>
+		<publisher>Sony</publisher>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="hbi-v1 video digitizer - sony.dsk" size="737280" crc="c039a66c" sha1="d720fa4142cac3a34d20dcd90c0273f751213dfa"/>
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx2_flop.xml:7150]: Is this actually working?  Doesn’t it require [a frame grabber cartridge](https://www.msx.org/wiki/Sony_HBI-V1) to actually do anything useful?

File: hash/msx2_flop.xml [hunk 2]
```diff
@@ -12215,6 +12288,33 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="sailorfk">
+		<description>Sailor Fuku Senshi Ferisu (Japan)</description>
+		<year>1990</year>
+		<publisher>Cocktail Soft</publisher>
+		<info name="alt_title" value="セーラー服戦士フェリス"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:12295]: Transliterated title should be “Sailor-fuku Senshi Felis”.

File: hash/msx2_flop.xml [hunk 3]
```diff
@@ -15471,6 +15748,30 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="yoshikmd2">
+		<description>Yoshida Koumuten Data Library Vol. 2 (Japan)</description>
+		<year>1990</year>
+		<publisher>MSX Magazine</publisher>
+		<info name="alt_title" value="吉田工務店データ集全３巻 VOL.2"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:15755]: Where does the Japanese title come from?  I haven’t seen a picture that shows that title.  The title screen shows “吉田工務店 Data Library Vol.2”.

File: hash/msx2_flop.xml [hunk 4]
```diff
@@ -15471,6 +15748,30 @@ The following floppies came with the machines.
...
+				<rom name="yoshida koumuten data library vol. 2 - msx magazine.dsk" size="737280" crc="559c3ce2" sha1="bc83532e77326bdecbe1818f3c15b5f5de572a45"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="yoshikmd3">
+		<description>Yoshida Koumuten Data Library Vol. 3 (Japan)</description>
+		<year>1990</year>
+		<publisher>MSX Magazine</publisher>
+		<info name="alt_title" value="吉田工務店データ集全３巻 VOL.3"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:15767]: Same for this one – where does the title come from?

File: hash/msx2_flop.xml [hunk 5]
```diff
@@ -13734,6 +13898,32 @@ The following floppies came with the machines.
...
+				<rom name="tokimeki sports gal - adult inn.dsk" size="737280" crc="a70b69b6" sha1="7f02a549ed4770b2ec912cb9d7049c16c1e144cf"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="tominaga">
+		<description>Tominaga Koukou Tantei-Bu (Japan)</description>
+		<year>1993</year>
+		<publisher>Team Plankton</publisher>
+		<info name="alt_tile" value="富永高校探偵部"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:13918]: If you’re going to be consistent with the other transliterated titles, it should be “Tominaga Koukou Tantei-bu”.

File: hash/msx2_flop.xml [hunk 6]
```diff
@@ -14010,6 +14200,51 @@ The following floppies came with the machines.
...
+				<rom name="the tucs - msx magazine.dsk" size="737280" crc="e0cdb9c0" sha1="b219f5236d1a20f9d9b32dae1b4c86c70faf6b72"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="tulipich">
+		<description>Tulip Ichigo (Japan)</description>
+		<year>1991</year>
+		<publisher>Tempin Software</publisher>
+		<info name="alt_title" value="チューリップ壱号"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:14239]: If you’re going to be consistent in indicating presence of long vowels, it should be “Tulip Ichigou” (or “Tulip Ichigō”, the MSX software lists seem to use a mixture).

File: hash/msx2_flop.xml [hunk 7]
```diff
@@ -15724,6 +16025,59 @@ The following floppies came with the machines.
...
+				<rom name="yumeji - fairytale (disk 3).dsk" size="737280" crc="e9b7b349" sha1="88ca21a4e75303ff10e9aa963d800195a4416535"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="yuureikun">
+		<description>Yuurei-Kun (Japan)</description>
+		<year>1989</year>
+		<publisher>System Sacom</publisher>
+		<info name="alt_title" value="幽霊君"/>
```
Comment (cuavas) [inline hash/msx2_flop.xml:16073]: Should be “Yuurei-kun” (or “Yūrei-kun”).

**Review by cuavas — COMMENTED** [review]
The transliterated Japanese titles in the MSX software lists seem to be inconsistent in whether they use ō/ū/etc. or ou/oo/uu/etc. to denote long vowels (and occasionally don’t denote them at all).  It would be nice if a consistent style were chosen.

**Comment by cuavas** [comment]
If someone else merges this, can you please apply any changes to descriptions and working status to the lists of additions when editing the commit message?  It really simplifies reconciling things for release notes.

File: hash/msx2_flop.xml (hunk 1 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:7150]: No, it isn't. I have updated the supported state.

File: hash/msx2_flop.xml (hunk 2 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:12295]: Thank you, I was unsure how to do the 'Ferisu' part so I had kept it as-is.

File: hash/msx2_flop.xml (hunk 3 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:15755]: That is how it was listed on generation-msx. But there is indeed no proof that the Japanese title ever was like that. I have changed it to match the title screen.

File: hash/msx2_flop.xml (hunk 4 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:15767]: Also from generation-msx. Updated to match the name displayed on the title screen.

File: hash/msx2_flop.xml (hunk 5 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:13918]: Updated to use Tantei-bu

File: hash/msx2_flop.xml (hunk 6 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:14239]: Updated the description.

File: hash/msx2_flop.xml (hunk 7 above)
Comment (wilbertpol) [reply hash/msx2_flop.xml:16073]: Updated the description

**Pull request merged**